views/manage_backlogOverview.mako

```diff
@@ -8,6 +8,7 @@
 %>
 <%block name="scripts">
 <script type="text/javascript" src="js/ajax-episode-search.js?${sbPID}"></script>
+<script type="text/javascript" src="js/manage/backlog-overview.js?${sbPID}"></script>
```

Comment:
This isn't needed as it's loaded in the main mako file and then the contents are run when the js loads.